### PR TITLE
5 patch api aticles article

### PR DESCRIPTION
**Pull request by sheffieldAndrew**
5 api/articles/aritcle_id

update votes on specific article

200s and 400s passed

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -128,3 +128,92 @@ describe("04 GET /api/articles/:article_id", () => {
...
+  test("400 - handles bad path - votes passed as string", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({ incl_votes: "hello" })
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Invalid - incl_votes must be a number");
+      });
+  });
+});
```
Comment (HeatherEver) [inline __tests__/app.test.js]: What if we were to pass in something with an incorrect key i.e inc_notes

**Review by HeatherEver — COMMENTED** [review]
Aamazing work Andrew but there's just one more point to think about

File: __tests__/app.test.js (hunk 1 above)
Comment (sheffieldAndrew) [reply __tests__/app.test.js]: ahahhh i see. no problem thanks, updated and resent

Comment (HeatherEver) [reply __tests__/app.test.js]: Thank youuu

**Review by HeatherEver — APPROVED** [review]
Niceee!

**Pull request merged**